src/modules/auth/auth.module.ts

```diff
@@ -68,6 +71,16 @@ export class AuthModule implements NestModule {
         { path: '/v1/login/siwe/initiate', method: RequestMethod.POST },
         { path: '/v1/login/siwe/verify', method: RequestMethod.POST }
       )
-      .forRoutes({ path: '/*', method: RequestMethod.ALL });
+      .forRoutes({ path: '/*', method: RequestMethod.ALL })
+      .apply((_req: Request, res: Response, next: NextFunction) => {
```

Comment:
`next` can be removed to fix warning
```suggestion
      .apply((_req: Request, res: Response) => {
```